app/utils/accountManagement.js

```diff
@@ -15,3 +15,11 @@ export async function registerAccount(accountName, displayName, profile) {
 
   return true;
 }
+
+export async function reviewAccount() {
```

Comment:
Review account sounds confusing to me. Can we rename that to getSelectedAccount or getCurrentAccount?